app/Services/Scheduler/EmailStatementService.php

```diff
@@ -92,6 +93,7 @@ private function calculateStartAndEndDates(): array
             EmailStatement::LAST_QUARTER => [now()->startOfDay()->subQuarterNoOverflow()->firstOfQuarter()->format('Y-m-d'), now()->startOfDay()->subQuarterNoOverflow()->lastOfQuarter()->format('Y-m-d')],
             EmailStatement::THIS_YEAR => [now()->startOfDay()->firstOfYear()->format('Y-m-d'), now()->startOfDay()->lastOfYear()->format('Y-m-d')],
             EmailStatement::LAST_YEAR => [now()->startOfDay()->subYearNoOverflow()->firstOfYear()->format('Y-m-d'), now()->startOfDay()->subYearNoOverflow()->lastOfYear()->format('Y-m-d')],
+            EmailStatement::ALL_TIME => [null, null],
```

Comment:
@turbo124 I think it would need to be the other way around, but also, the date is shown in the email and PDF templates. I added conditions to either hide the date range or show something else reasonable based on the invoices that were returned. If we set the dates here, then the email and PDF would show that the statement is from January 1, 1970 to now. Maybe that's OK though.

Reply:
I think the preferable approach would be to calculate the first and last invoice dates and then use this as the range. saves a lot of changes downstream also.